Sources/Extensions/CoreGraphics/CGVectorExtensions.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  CGVectorExtensions.swift
+//  SwifterSwift
+//
+//  Created by Robbie Moyer on 7/25/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if canImport(CoreGraphics)
+import CoreGraphics
+
+// MARK: - Properties
+public extension CGVector {
+
+    /// SwifterSwift: The angle of rotation (in radians) of the vector.
+    /// The range of the angle is -π to π; an angle of 0 points to the right.
+    public var angle: CGFloat {
+        return atan2(dy, dx)
+    }
+
+    /// SwifterSwift: The magnitude (or length) of the vector.
+    public var magnitude: CGFloat {
+        return sqrt(dx*dx + dy*dy)
```

Comment:
Small style comment, but I think there should always be spacing between operators (except prefix `-`). I have the feeling that you omitted it here to show the grouping, but in that case I think it should be done with parentheses, even though they're not really necessary.
`sqrt((dx * dx) + (dy * dy))`